ui/app/models/kv/metadata.js

```diff
@@ -44,7 +45,7 @@ export default class KvSecretMetadataModel extends Model {
     editType: 'ttl',
     label: 'Automate secret deletion',
     helperTextDisabled: `A secret's version must be manually deleted.`,
-    helperTextEnabled: 'Delete all new versions of this secret after.',
+    helperTextEnabled: 'Delete all new versions of this secret after:',
```

Comment:
I know... removing a period. But in this case it makes sense.
![image](https://github.com/hashicorp/vault/assets/6618863/b4e0271a-5206-4149-92c4-2a2a41d7ca42)